hnn_core/network.py

```diff
@@ -1206,7 +1209,7 @@ def add_connection(self, src_gids, target_gids, loc, receptor,
             gid_pairs[src_gid] = target_src_pair
 
         conn['src_type'] = self.gid_to_type(src_gids[0])
-        conn['src_gids'] = list(set(src_gids))
+        conn['src_gids'] = sorted(set(src_gids))
```

Comment:
```suggestion
        conn['src_gids'] = list(sorted(set(src_gids)))
```

to keep the datatype same as before?

Reply:
`sorted` returns a list!